env.js

```diff
@@ -39,6 +41,7 @@ module.exports = {
         BASE_URL,
         CORS_PROXY_PREFIX,
         NODE_ENV,
-        STATUS_URL
+        STATUS_URL,
```

Comment:
Two small things:
1. These are sorted alphabetically, would prefer if you can keep that.
2. It's not documented, but the way this object works is that items in `processEnv` will be sent to the client, and items outside of it are available on the server. I think this new variable should be in both places, so can you add it at both levels of the object?